app/routes/search/agendaitems.js

```diff
@@ -62,6 +62,10 @@ export default class AgendaitemSearchRoute extends Route {
     if (entry.title && Array.isArray(entry.title)) {
       entry.title = entry.title.join(' ');
     }
+    if ((agendaitem.highlight && agendaitem.highlight.shortTitle) && agendaitem.highlight.shortTitle[0]?.split(" ").length < agendaitem.attributes.shortTitle?.split(" ").length) {
+      const highlightShortTitleEmsRemoved = agendaitem.highlight.shortTitle[0].replace(/<em>|<\/em>/g, "");
+      entry.shortTitle = agendaitem.attributes.shortTitle.replace(highlightShortTitleEmsRemoved, agendaitem.highlight.shortTitle[0]);
```

Comment:
change this to `replaceAll` and Sven's comment should be addressed

Reply:
correction: this should be in a for loop, since we're replacing the whole highlight snippet